docker-compose.yml

```diff
@@ -36,9 +54,16 @@ services:
     depends_on:
      - config-server
      - discovery-server
-    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s","--","java", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseCGroupMemoryLimitForHeap", "-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
+     - database-service
```

Comment:
dobre, ale lepsi by bylo mit pro kazdou sluzbu oddelenou databazi

Reply:
Pro ukazku, jak by se to dalo udelat, jsem udelal ukazku na jednem databazovem serveru, jak by se to dalo prekonfigurovat, aby nebyla pouzita in-memory databaze.

Reply:
rozumim, ja jenom poukazoval na to, jak by se to resilo v realnem provozu